config/sync/config_ignore.settings.yml

```diff
@@ -4,6 +4,7 @@ ignored_config_entities:
   - 'block.block.*'
   - '~block.block.seven_*'
   - '~block.block.stanford_basic_*'
+  - '~block.block.config_pages*'
```

Comment:
I think we should rename the blocks to be `stanford_basic_` prefixed. it will help keep things consistent. And when a subtheme is enabled, it creates new block configs with the theme name prefixed as well.

Reply:
Yeah, I like that. Will do.